packages/app/src/vis-packs/core/raw/RawVisContainer.tsx

```diff
@@ -11,7 +11,7 @@ function RawVisContainer(props: VisContainerProps) {
   assertNonNullShape(entity);
 
   return (
-    <VisBoundary resetKey={entity}>
+    <VisBoundary>
```

Comment:
Just a quick thing I noticed while looking at our usage of `ErrorBoundary`. There's no need for this `resetKey` prop here, since when `entity` changes, the path changes and therefore the whole subtree is unmounted/remounted. In other components, we pass either `dimMapping` or nothing like here when there's no dimension mapper.